docs/docs/functions/macros.md

```diff
@@ -39,11 +39,34 @@ Defines new blocks that can be inserted into an existing chain.
 
 Defines a new chain that isn't looped.
 
+Such a chain executes only once when invoked (via `run <node>`).
+
 === "Code"
 
     ```clojure linenums="1"
+    ;; define a node (main)
+    (defnode main)
+    ;; define a non-looped chain (chain-hello)
+    (defchain chain-hello
+        (Msg "Hello World!"))
+    ;; define another non-looped chain (chain-bye)
+    (defchain chain-bye
+        (Msg "Goodbye World"))
+    ;; schedule the non-looped chains on the node
+    (schedule main chain-hello)
+    (schedule main chain-bye)
+    ;; run all the scheduled chains on the node
+    (run main)
+    ;; Even if provided, the FPS and iteration parameters are ignored for non-looped chains
+    ;; (run main 0.02 5)
```

Comment:
@sinkingsugar is it correct stating it this way? For instance, if a non-looped chain has a `(Recur)` block, would that be impacted by the delay parameter?

Reply:
this is not correct, non looped (main, e.g. defchain) chains still can yield, and so they still can iterate, and this determines the iteration speed/fps

Reply:
This description of the `run` parameters is moved out of `defchain` and into `run`.